README.md

```diff
@@ -8,6 +8,8 @@ VSCode extension (plugin) that supports the Ion data format.
 ## Functionality 
 
 * Syntax highlighting. 
+* Error reporting. 
+* Formatting. 
 
 ## Development 
 
```

Comment:
We should also add the instructions on how to check out the repo. You can reuse the text and from one of our other libs here.